e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -55,6 +55,51 @@ describe('Integrations', () => {
         });
     });
 
+    it('MM-T662 /join command for private channels', () => {
+        const privateChannelName = user1.username;
+
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl1);
+
+        // # User 1 Create a private channel, with ${channelName}
+        cy.get('#createPrivateChannel').click();
+        cy.get('#private').click();
+        cy.get('#newChannelName').type(privateChannelName);
+        cy.get('#submitNewChannel').click();
+
+        // # User with privilege try join command without tilde
+        cy.get('#publicChannelList').findByText('Town Square').click();
+        cy.postMessage(`/join ${privateChannelName}`);
+
+        // * private channel should be active
```

Comment:
nit: Please capitalize comment
```suggestion
        // * Private channel should be active
```